packages/relay/test/report.test.ts

```diff
@@ -190,7 +191,7 @@ describe('POST /api/report', function () {
             (post) => post.postId === postId
         )
         expect(reportedPost).to.exist
-        expect(reportedPost).to.not.have.property('content')
+        expect(reportedPost).to.have.property('content', 'test content')
```

Comment:
I think this is a typo?

Reply:
this is a response, it should be content